deploy/helm/sumologic/templates/otelcol-deployment.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if eq .Values.sumologic.traces.enable true }}
+apiVersion: extensions/v1beta1
```

Comment:
should probably update this as this api is gone in 1.16

Reply:
Fixed.